src/providers/citation.ts

```diff
@@ -35,10 +35,32 @@ export class Citation {
             this.citationInBib[bibPath].forEach(item => items.push(item))
         })
 
+        const configuration = vscode.workspace.getConfiguration('latex-workshop')
         this.suggestions = items.map(item => {
             const citation = new vscode.CompletionItem(item.key, vscode.CompletionItemKind.Reference)
             citation.detail = item.title
-            citation.filterText = `${item.author} ${item.title} ${item.journal}`
+            switch (configuration.get('citation_intellisense_label') as string) {
+                case 'bibtex key':
+                default:
+                    citation.label = item.key
+                    break
+                case 'title':
+                    if (item.title) {
```

Comment:
I think here we should set
```ts
citation.detail = undefined
```
so that the title is not clipped by vscode trying to show the title again in the detail field..